Aptoide/aptoide-client-v8
# [MOB-462] User Feedback events

**What does this PR do?**

This PR aims at implementing user feedback events for vote up and down in comments, reactions, post comment, reply to another comment (in all contexts, app and store).

**Database changed?**

 No

**Where should the reviewer start?**

- [x] UserFeedbackAnalytics.kt

**How should this be manually tested?**

Open the app, perform the interactions that would trigger the events (post comments, reply to comments, add reactions, upvote and downvote comments) and check that the events are being correctly sent to Facebook.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-462](https://aptoide.atlassian.net/browse/MOB-462)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
